### PR TITLE
ACS-6374: create event listener

### PR DESCRIPTION
**Pull request by Pawel-608**

### CODE REVIEW

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/LiveIngesterRouteBuilder.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    public LiveIngesterRouteBuilder(CamelContext context, ActiveMQProperties activeMQProperties)
+    {
+        super(context);
+        this.properties = activeMQProperties;
+    }
+
+    public void configure()
+    {
+        System.out.println(properties);
```
Comment (tpage-alfresco) [inline live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/LiveIngesterRouteBuilder.java:36]: I guess we should get rid of debugging before merging this.

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/config/ActiveMQProperties.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import lombok.ToString;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+@Getter
+@Setter
+@ToString
+@Validated
+@ConfigurationProperties(prefix = "alfresco.ingester.messaging")
+public class ActiveMQProperties
```
Comment (tpage-alfresco) [inline live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/config/ActiveMQProperties.java:15]: and this file is missing the copyright header completely.

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/LiveIngesterRouteBuilder.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+import static org.apache.camel.LoggingLevel.DEBUG;
+
+import lombok.extern.slf4j.Slf4j;
+import org.alfresco.hxi_connector.live_ingester.routes.config.ActiveMQProperties;
+import org.apache.camel.CamelContext;
+import org.apache.camel.builder.RouteBuilder;
+import org.springframework.stereotype.Component;
+
+@Slf4j
```
Comment (tpage-alfresco) [inline live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/LiveIngesterRouteBuilder.java:21]: Is this used?

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/LiveIngesterRouteBuilder.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    public LiveIngesterRouteBuilder(CamelContext context, ActiveMQProperties activeMQProperties)
+    {
+        super(context);
+        this.properties = activeMQProperties;
+    }
+
+    public void configure()
+    {
+        System.out.println(properties);
+        from(properties.getChannel()).transacted().routeId("ingester-events-consumer").log(DEBUG, "Received path event : ${header.JMSMessageID}").end();
```
Comment (Pawel-608) [inline live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/LiveIngesterRouteBuilder.java:37]: can we somehow setup clang format to not compress it to one line?

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/config/ActiveMQProperties.java (hunk 2 above)
Comment (tpage-alfresco) [reply live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/config/ActiveMQProperties.java:15]: Probably the class shouldn't mention ActiveMQ - it's `MessageQueueProperties` or something.

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/LiveIngesterRouteBuilder.java [hunk 5]
```diff
@@ -0,0 +1,39 @@
+/*-
+ * #%L
+ * %%
+ * Copyright (C) 2020 Alfresco Software Limited
+ * %%
+ * License rights for this program may be obtained from Alfresco Software, Ltd.
+ * pursuant to a written agreement and any use of this program without such an
+ * agreement is prohibited.
```
[the remark this answers is not in the document]
Comment (Pawel-608) [reply live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/LiveIngesterRouteBuilder.java]: fixed

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/LiveIngesterRouteBuilder.java (hunk 1 above)
Comment (Pawel-608) [reply live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/LiveIngesterRouteBuilder.java]: removed

**Review by tpage-alfresco — COMMENTED** [review]
A few minor things to investigate.

Can we add a test without using a real Repository? Maybe it doesn't make sense until we're producing output in a more testable location?

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/config/ActiveMQProperties.java (hunk 2 above)
Comment (Pawel-608) [reply live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/config/ActiveMQProperties.java:41]: added

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/LiveIngesterRouteBuilder.java (hunk 3 above)
Comment (Pawel-608) [reply live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/routes/LiveIngesterRouteBuilder.java]: nope

**Pull request merged**